WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -579,36 +581,15 @@ - (void)showViewAdmin
 
     [WPAppAnalytics track:WPAnalyticsStatOpenedViewAdmin withBlog:self.blog];
 
-    NSString *dashboardUrl = [self.blog.xmlrpc stringByReplacingOccurrencesOfString:@"xmlrpc.php" withString:@"wp-admin/"];
+    NSString *dashboardUrl;
+    if (self.blog.isHostedAtWPcom) {
+        dashboardUrl = [NSString stringWithFormat:@"https://wordpress.com/stats/insights/%@", self.blog.hostname];
```

Comment:
I don't feel too happy about hardcoding a URL with `stats` in it as the "dashboard". I'd rather have a url in calypso of the form `https://wordpress.com/site/%@` that redirected to whatever was the default page to manage a site.